system/include/webgl/webgl1_ext.h

```diff
@@ -90,61 +128,91 @@ WEBGL_APICALL void GL_APIENTRY emscripten_webgl_getTranslatedShaderSource(GLuint
 #define GL_MAX_DRAW_BUFFERS_WEBGL 0x8824
 WEBGL_APICALL void GL_APIENTRY emscripten_glDrawBuffersWEBGL(GLsizei n, const GLenum *buffers);
 WEBGL_APICALL void GL_APIENTRY glDrawBuffersWEBGL(GLsizei n, const GLenum *buffers);
+#endif /* EMSCRIPTEN_GL_WEBGL_draw_buffers */
 
 // 19. https://www.khronos.org/registry/webgl/extensions/ANGLE_instanced_arrays/
+#ifndef EMSCRIPTEN_GL_ANGLE_instanced_arrays
+#define EMSCRIPTEN_GL_ANGLE_instanced_arrays 1
 #define GL_VERTEX_ATTRIB_ARRAY_DIVISOR_ANGLE 0x88FE
 WEBGL_APICALL void GL_APIENTRY emscripten_glDrawArraysInstancedANGLE(GLenum mode, GLint first, GLsizei count, GLsizei primcount);
 WEBGL_APICALL void GL_APIENTRY emscripten_glDrawElementsInstancedANGLE(GLenum mode, GLsizei count, GLenum type, GLintptr offset, GLsizei primcount);
 WEBGL_APICALL void GL_APIENTRY emscripten_glVertexAttribDivisorANGLE(GLuint index, GLuint divisor);
 WEBGL_APICALL void GL_APIENTRY glDrawArraysInstancedANGLE(GLenum mode, GLint first, GLsizei count, GLsizei primcount);
 WEBGL_APICALL void GL_APIENTRY glDrawElementsInstancedANGLE(GLenum mode, GLsizei count, GLenum type, GLintptr offset, GLsizei primcount);
 WEBGL_APICALL void GL_APIENTRY glVertexAttribDivisorANGLE(GLuint index, GLuint divisor);
+#endif /* EMSCRIPTEN_GL_ANGLE_instanced_arrays */
 
 // 20. https://www.khronos.org/registry/webgl/extensions/OES_texture_float_linear/
+#ifndef EMSCRIPTEN_GL_OES_texture_float_linear
+#define EMSCRIPTEN_GL_OES_texture_float_linear 1
 // <no symbols exposed>
+#endif /* EMSCRIPTEN_GL_OES_texture_float_linear */
 
 // 21. https://www.khronos.org/registry/webgl/extensions/OES_texture_half_float_linear/
+#ifndef EMSCRIPTEN_GL_OES_texture_half_float_linear
+#define EMSCRIPTEN_GL_OES_texture_half_float_linear 1
 // <no symbols exposed>
+#endif /* EMSCRIPTEN_GL_OES_texture_half_float_linear */
 
 // 25. https://www.khronos.org/registry/webgl/extensions/EXT_blend_minmax/
+#ifndef EMSCRIPTEN_GL_EXT_blend_minmax
+#define EMSCRIPTEN_GL_EXT_blend_minmax 1
 #define GL_MIN_EXT 0x8007
 #define GL_MAX_EXT 0x8008
+#endif /* EMSCRIPTEN_GL_EXT_blend_minmax */
 
 // 27. https://www.khronos.org/registry/webgl/extensions/EXT_shader_texture_lod/
+#ifndef EMSCRIPTEN_GL_EXT_shader_texture_lod
+#define EMSCRIPTEN_GL_EXT_shader_texture_lod 1
 // <no symbols exposed>
-
-// 12. https://www.khronos.org/registry/webgl/extensions/WEBGL_compressed_texture_atc/
-#define GL_COMPRESSED_RGB_ATC_WEBGL 0x8C92
-#define GL_COMPRESSED_RGBA_ATC_EXPLICIT_ALPHA_WEBGL 0x8C93
-#define GL_COMPRESSED_RGBA_ATC_INTERPOLATED_ALPHA_WEBGL 0x87EE
```

Comment:
what happened with these three deleted defines?

Reply:
https://github.com/emscripten-core/emscripten/pull/12386#discussion_r497857721

Would you prefer we didn't do this or do this separately?

Reply:
Let's indeed drop the rejected extensions, best to get compile time errors about things never being able to work out. Ok with that change landing in this PR.